deps/python-kombu/python-kombu.spec

```diff
@@ -11,7 +11,7 @@ Name:           python-%{srcname}
 # The Fedora package is using epoch 1, so we need to also do that to make sure ours gets installed
 Epoch:          1
 Version:        3.0.24
-Release:        5.pulp%{?dist}
+Release:        7.pulp%{?dist}
```

Comment:
`python-kombu-3.0.24-6.pulp` is already taken and koji already has it built and locked. The `7` release is the next available one for `3.0.24`. The `7` release also includes all the same stuff from `6`.